bioblend/galaxy/users/__init__.py

```diff
@@ -145,3 +167,21 @@ def delete_user(self, user_id, purge=False):
         if purge is True:
             params['purge'] = purge
         return self._delete(id=user_id, params=params)
+
+    def get_user_apikey(self, user_id):
+        """
+        Get the current API key for a given user.
+
+        :type user_id: str
+        :param user_id: encoded user ID
+
+        :rtype: str
+        :return: the API key for the user
+        """
+
+        url = self.gi._make_url(self, None)
+        url = '/'.join([url, user_id, 'api_key', 'inputs'])
+        payload = {}
+        payload['user_id'] = user_id
```

Comment:
@erasche I don't think the `payload` is needed here, I'm going to remove it. Let me know if I'm missing something.